### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by ghost**
Hi
is this commit ok ?
thanks for your great work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ func main() {
     cfg := &client.Config{
         Host:         "http://localhost:8080",
         DisableCache: false, // defaults to false, here for documentation
+      	Header: make(map[string][]string),
```
Comment (nilslice) [inline README.md]: Could you use the built-in `http.Header` here instead?

**Comment by nilslice** [comment]
You're welcome -- glad you are finding Ponzu useful. Thank you for the PR, I appreciate your help! See my comment on your example in the README, happy to discuss.

**Comment by nilslice** [comment]
Thanks -- if you don't mind sharing, what are you using the Go client for?

**Comment by ghost** [comment]
Hi nilslice:
I found ponzu in the internet, it seems great.
I  was searching for this kind of software for speeding up the development of API server.

**Comment by nilslice** [comment]
@neontx - thank you, and I apologize for the delay!

**Pull request merged**